Sources/OktaAnalytics/OktaAnalytics.swift

```diff
@@ -42,8 +45,12 @@ public final class OktaAnalytics: NSObject {
 
      - Parameters:
         - securityAppGroupIdentifier: shared group Identifier to be data shared between app, extensions, app and widgets.
+        - expirationPeriodForScenarioEvents: expiration period for scenario events.
      */
-    public static func initializeStorageWith(securityAppGroupIdentifier: String) {
+    public static func initializeStorageWith(securityAppGroupIdentifier: String, expirationPeriodForScenarioEvents: UInt? = nil) {
```

Comment:
Why we need `expirationPeriodForScenarioEvents ` as a parameter? It is just required only in one public API - `reportAndDeleteExpiredEvents`
Alternatively, expiration can be a property of the `ScenarioEvent` object itself

Reply:
The idea is to set `expirationPeriod` one time when we initialize OktaAnalytics. 
It will guarantee that this interval will not change and we always have the same period of time for expired events. And this parameter can be changed in OV if we decide to increase or decrease it.

Reply:
I still don't understand. It is being used only in one function `reportAndDeleteExpiredEvents`, why just no make it as parameter with default value?
We don't need extra property for the instance. Less stateful properties is always good